src/meta/typesmap.go

```diff
@@ -36,11 +36,15 @@ func (m RawTypesMap) Add(typ Type) {
 	m[typ] = struct{}{}
 }
 
+func (m RawTypesMap) Delete(typ Type) {
```

Comment:
It does not make too much sense to me to declare methods like "Delete" and "Add" to value types

Reply:
Thanks for the comments!
Perhaps here Delete is too much, but `res.Add("some")` looks simpler than `res["some"] = struct{}{}`, that's why I introduced it.
Delete is just for a similar interface.